app/middleware/auth.js

```diff
@@ -0,0 +1,57 @@
+const jwt = require('jsonwebtoken');
+
+/*
+exports.verifyToken = (req, res, next) => {
+	try {
+		const token = req.headers.authorization.split('Bearer ')[1];
+		let jwt_secret = 'jwt-secret-key';
+
+		req.decoded = jwt.verify(token, jwt_secret);
+		return next();
+	} catch (e) {
+		if (e.name === 'TokenExpiredError') {
+			return res.status(419).json({
+				success: false,
+				message: 'Token 만료'
+			});
+		}
+		return res.status(401).json({
+			success: false,
+			message: 'Token이 유효하지 않습니다'
+		});
+	}
+}
+
+*/
+
+exports.verifyToken = (req, res, next) => {
+	const token = req.headers['x-access-token'] || req.query.token
```

Comment:
Codacy has a fix for the issue: [Replace `'x-access-token']·||·req.query.token` with `"x-access-token"]·||·req.query.token;`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
	const token = req.headers["x-access-token"] || req.query.token;
```